lightkurve/targetpixelfile.py

```diff
@@ -537,3 +543,195 @@ def get_bkg_lightcurve(self, aperture_mask=None):
+        cols.append(fits.Column(name='FLUX_ERR', format=eformat,
+                                unit='e-/s', dim=coldim,
+                                array=self.flux_err))
+        cols.append(fits.Column(name='FLUX_BKG', format=eformat,
+                                unit='e-/s', dim=coldim,
+                                array=self.flux_bkg))
+        cols.append(fits.Column(name='FLUX_BKG_ERR', format=eformat,
+                                unit='e-/s', dim=coldim,
+                                array=self.flux_bkg_err))
+        cols.append(fits.Column(name='COSMIC_RAYS', format=eformat,
+                                unit='e-/s', dim=coldim,
+                                array=self.cosmic_rays))
+        cols.append(fits.Column(name='QUALITY', format='J',
+                                array=self.quality))
+        cols.append(fits.Column(name='POS_CORR1', format='E', unit='pixels',
+                                array=self.pos_corr1))
+        cols.append(fits.Column(name='POS_CORR2', format='E', unit='pixels',
+                                array=self.pos_corr2))
+        coldefs = fits.ColDefs(cols)
+        hdu = fits.BinTableHDU.from_columns(coldefs)
+
+        # Set the header with defaults
+        template = self._header_template(1)
+        for kw in template:
+            if kw not in ['XTENSION', 'NAXIS1', 'NAXIS2', 'CHECKSUM', 'BITPIX']:
+                try:
+                    hdu.header[kw] = (self.keywords[kw],
+                                      self.keywords.comments[kw])
+                except KeyError:
+                    hdu.header[kw] = (template[kw],
+                                      template.comments[kw])
+
+        # Override the defaults where necessary
+        hdu.header['EXTNAME'] = 'TARGETTABLES'
+        hdu.header['OBJECT'] = self.target_id
+        hdu.header['KEPLERID'] = self.target_id
+        for n in [5, 6, 7, 8, 9]:
+            hdu.header["TFORM{}".format(n)] = eformat
+            hdu.header["TDIM{}".format(n)] = coldim
+        hdu.header['TFORM4'] = jformat
+        hdu.header['TDIM4'] = coldim
+
+        return hdu
+
+    def _make_aperture_extension(self):
+        """Create the aperture mask extension."""
+        mask = 3 * np.ones((self.n_rows, self.n_cols), dtype='int32')
+        hdu = fits.ImageHDU(mask)
+
+        # Set the header from the template TPF again
+        template = self._header_template(2)
+        for kw in template:
+            if kw not in ['XTENSION', 'NAXIS1', 'NAXIS2', 'CHECKSUM', 'BITPIX']:
+                try:
+                    hdu.header[kw] = (self.keywords[kw],
+                                      self.keywords.comments[kw])
+                except KeyError:
+                    hdu.header[kw] = (template[kw],
+                                      template.comments[kw])
+        hdu.header['EXTNAME'] = 'APERTURE'
+        return hdu
+
+
+def create_tpf_hdu(images, position, size=(10, 10), target_id="unnamed-target"):
```

Comment:
Is `images` a numpy array, fits files or a list of file names? I think it's a list of file names, could we make it a more descriptive name? I would have read that as a numpy array of values or a list of astropy fits objects.